tests/SIL.LCModel.Tests/TestData/DataMigration7000072.xml

```diff
@@ -0,0 +1,804 @@
+</Status>
+<Styles>
+<objsur guid="164123a0-ca12-4c44-bd69-d5fb9d4b1a0b" t="o" />
+<objsur guid="1b25e2a0-5f78-40d1-943a-2b14caccf860" t="o" />
+<objsur guid="1fafdfa6-34a7-4577-b4aa-0c6518af623a" t="o" />
+<objsur guid="23efa4cc-0347-4fc1-a33d-57b57ee852f9" t="o" />
+<objsur guid="25d17a96-4130-49c5-8f2b-497c1953556a" t="o" />
+<objsur guid="282f9ae7-d4dc-4c9b-93b5-413a7d3986b1" t="o" />
+<objsur guid="3191611c-41c6-4ecf-bcd5-a5084cc3e1f2" t="o" />
+<objsur guid="31d215a7-ac39-4d1c-be7d-88fe2b7c8d34" t="o" />
+<objsur guid="366e9470-26f7-4bdf-86e4-f829784c3113" t="o" />
+<objsur guid="3c8c511b-4683-4289-88d0-024f66540f6f" t="o" />
+<objsur guid="3eb5354a-9f55-4165-a411-eca7f9d2a845" t="o" />
+<objsur guid="3f844794-09d1-45fa-9d75-05c62e2b836d" t="o" />
+<objsur guid="4d6cffc4-254e-4ef7-b26b-41aab4bfbf48" t="o" />
+<objsur guid="501c3363-065d-465a-baad-15d0787f9ded" t="o" />
+<objsur guid="693aedeb-1f1a-4f1f-b6cd-4a2a96a1dc37" t="o" />
+<objsur guid="6de6e4d1-0c06-4102-86e2-82faf7bf9811" t="o" />
+<objsur guid="6ed2cb3d-f6db-4af5-9de9-75669ce26f46" t="o" />
+<objsur guid="740c5846-9d09-4109-806c-5357fede8712" t="o" />
+<objsur guid="7434c828-a430-4084-8a49-4be3bfa97068" t="o" />
+<objsur guid="77997501-47d3-4574-ae1d-7300107dfb73" t="o" />
+<objsur guid="7a194e30-4c8c-4934-83ad-b12ecc47a768" t="o" />
+<objsur guid="7d2c80cd-af87-4d56-8337-cfd389b775bb" t="o" />
+<objsur guid="7ff691e7-32f4-445d-acea-1ded6a6ac8ea" t="o" />
+<objsur guid="80a47ed1-1e29-40d4-9aa0-d3041a8ca9dc" t="o" />
+<objsur guid="879d3c6f-8e62-4887-a4af-27feb88c2af1" t="o" />
+<objsur guid="89670d4f-71d4-4878-b2ff-815e45512be1" t="o" />
+<objsur guid="905a485e-4b54-47eb-a8e9-1f86e9c25699" t="o" />
+<objsur guid="90b94e23-3e77-4460-94fd-c1997ed9a81f" t="o" />
+<objsur guid="9132811f-a2e6-43b1-92f2-6e2030bbea12" t="o" />
+<objsur guid="92e5dc56-317a-461d-a179-e9e005d6612a" t="o" />
+<objsur guid="945e6502-615b-4863-a18c-111e4d6b36c6" t="o" />
+<objsur guid="9795fb02-12d8-4d1e-9546-5f1bf1c848ea" t="o" />
+<objsur guid="9ece8f19-30f0-43a0-bf7b-9e5f33c6cea7" t="o" />
+<objsur guid="9fa82bd0-e797-4c40-88d7-cb591613b426" t="o" />
+<objsur guid="aba229f3-ca42-4727-a903-0abc3d418607" t="o" />
+<objsur guid="cb0fbdb3-2ca2-473b-97bb-8f505e18010c" t="o" />
+<objsur guid="d80764cc-4d86-4e14-b217-d1b0a4dc3379" t="o" />
+<objsur guid="dd807cae-4b18-4354-934d-99632b663f9e" t="o" />
+<objsur guid="f41be047-a4ee-402b-bcfe-9458e502e682" t="o" />
+<objsur guid="fb6450f5-626d-4fbf-ada5-4cfd5986b5b0" t="o" />
+</Styles>
+<TextMarkupTags>
+<objsur guid="c1af47dc-ea5e-11de-8e34-0013722f8dec" t="o" />
+</TextMarkupTags>
+<TimeOfDay>
+<objsur guid="d850bdae-ea5e-11de-985e-0013722f8dec" t="o" />
+</TimeOfDay>
+<TranslationTags>
+<objsur guid="d7f71649-e8cf-11d3-9764-00c04f186933" t="o" />
+</TranslationTags>
+<VernWss>
+<Uni>fr</Uni>
+</VernWss>
+</rt>
+<rt class="LexDb" guid="af26d792-ea5e-11de-8f7e-0013722f8dec" ownerguid="766e2ecf-7dde-42a4-8a8c-66432fc81e88">
+<ComplexEntryTypes>
```

Comment:
Remove everything from this element except the ReversalIndexes element and Dates